lib/instagram.js

```diff
@@ -1441,20 +1441,20 @@ var instagram = function(spec, my) {
   };
 
   /**
-   * Retrieves information about a given media based on its url
-   * @param url string the url of the media
+   * Retrieves information about a given media based on its url (see http://instagram.com/developer/embedding/)
+   * @param parameters object {url, callback, omitscript, hidecaption, maxwidth}
    * @param cb function (err, result, limit);
    */
-  oembed = function(url, cb) {
+  oembed = function(params, cb) {
```

Comment:
I'd prefer to use the following signature: `function(url, options, cb)` since the url is always specified, but not the options. It'll also avoid a change in the API.

You can have a look at methods like `user_media_recent` for an example of the use of `options` and making it optional.  `params` should be constructed from the `options` object.